swagger.yaml

```diff
@@ -4734,3 +4734,30 @@ definitions:
         type: string
         enum: [beta, production]
         description: Level of support of the version.
+
+  operator-property:
```

Comment:
This is a great start.  I actually designed a similar API in the past, and it came out almost the same.  Obviously we may want to expand it in the future as needed.  We may need a "mandatory" boolean to indicate if a property is required or not.

Reply:
Added.